clusterdock/cli.py

```diff
@@ -56,6 +56,21 @@ def main():
     build_parser = action_subparsers.add_parser('build',
                                                 formatter_class=FORMATTER_CLASS,
                                                 add_help=False)
+    build_parser.add_argument('--namespace',
+                              help='Namespace to use when looking for images',
```

Comment:
Is this for looking for images or for naming them?

Reply:
as discussed offline, removed this and now have 'repository' arg